app/src/main/java/com/pragati/karuna/util/Extensions.kt

```diff
@@ -0,0 +1,60 @@
+package com.pragati.karuna.util
+
+import android.graphics.drawable.Drawable
+import android.view.View
+import android.widget.TextView
+
+fun TextView.setLeftDrawable(drawable: Drawable?) {
+    this.setCompoundDrawablesWithIntrinsicBounds(
+        drawable,
+        null,
+        null,
+        null
+    )
+}
+
+
+fun View.enable() {
+    alpha = 1f
+    this.isEnabled = true
+}
+
+fun View.disable() {
+    alpha = 0.5f
+    this.isEnabled = false
+}
+
+/**
+ * Extension method to provide show keyboard for View.
+ */
```

Comment:
I think this comment is a small mistake! is it?

Reply:
Yes. I copy pasted from other project.. I will remove this..